cmd/kubelet/app/plugins.go

```diff
@@ -47,12 +47,15 @@ func ProbeVolumePlugins() []volume.VolumePlugin {
 	// The list of plugins to probe is decided by the kubelet binary, not
 	// by dynamic linking or other "magic".  Plugins will be analyzed and
 	// initialized later.
+	//
+	// Kubelet does not currently need to configure volume plugins.
+	// If/when it does, see kube-controller-manager/app/plugins.go for example of using volume.VolumeConfig
 	allPlugins = append(allPlugins, aws_ebs.ProbeVolumePlugins()...)
 	allPlugins = append(allPlugins, empty_dir.ProbeVolumePlugins()...)
 	allPlugins = append(allPlugins, gce_pd.ProbeVolumePlugins()...)
 	allPlugins = append(allPlugins, git_repo.ProbeVolumePlugins()...)
-	allPlugins = append(allPlugins, host_path.ProbeVolumePlugins()...)
-	allPlugins = append(allPlugins, nfs.ProbeVolumePlugins()...)
+	allPlugins = append(allPlugins, host_path.ProbeVolumePlugins(volume.VolumeConfig{})...)
+	allPlugins = append(allPlugins, nfs.ProbeVolumePlugins(volume.VolumeConfig{})...)
```

Comment:
@pmorie The implementation would run thusly, in accordance with this PR:

1.  add VolumeConfigFlags to the Kubelet binary and enumerate all CLI flags.
  1. example: FSRelabelSupportedNFS, FSRelabelEnabledNFS
2.  add the common property to volume.VolumeConfig
  1. example: FSRelabelSupported and FSRelabelEnabled
3.  Map CLI flag to VolumeConfig in Kubelet's app/plugins.go
  1.  See example: https://github.com/markturansky/kubernetes/blob/13065f0b0402175729492fadec31b3f82a76388e/cmd/kube-controller-manager/app/plugins.go#L34
4.  And the rest of the plumbing in the plugins themselves